package.json

```
@@ -15,60 +15,60 @@
   },
   "dependencies": {
-    "@angular/animations": "14.0.0",
-    "@angular/common": "14.0.0",
-    "@angular/compiler": "14.0.0",
-    "@angular/core": "14.0.0",
-    "@angular/forms": "14.0.0",
-    "@angular/platform-browser": "14.0.0",
-    "@angular/platform-browser-dynamic": "14.0.0",
-    "@angular/router": "14.0.0",
-    "@nrwl/angular": "14.2.0-rc.2",
-    "rxjs": "7.4.0",
+    "@angular/animations": "14.0.2",
+    "@angular/common": "14.0.2",
+    "@angular/compiler": "14.0.2",
+    "@angular/core": "14.0.2",
+    "@angular/forms": "14.0.2",
+    "@angular/platform-browser": "14.0.2",
+    "@angular/platform-browser-dynamic": "14.0.2",
+    "@angular/router": "14.0.2",
+    "@nrwl/angular": "14.3.6",
+    "rxjs": "7.5.5",
     "tslib": "2.3.0",
-    "zone.js": "0.11.4"
+    "zone.js": "0.11.6"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "14.0.0",
-    "@angular-eslint/eslint-plugin": "13.2.1",
-    "@angular-eslint/eslint-plugin-template": "13.2.1",
-    "@angular-eslint/template-parser": "13.2.1",
-    "@angular/cli": "14.0.0",
-    "@angular/compiler-cli": "14.0.0",
-    "@angular/language-service": "14.0.0",
-    "@nrwl/cli": "14.2.0-rc.2",
-    "@nrwl/cypress": "14.2.0-rc.2",
-    "@nrwl/eslint-plugin-nx": "14.2.0-rc.2",
-    "@nrwl/jest": "14.2.0-rc.2",
-    "@nrwl/linter": "14.2.0-rc.2",
-    "@nrwl/nx-cloud": "14.0.8",
-    "@nrwl/workspace": "14.2.0-rc.2",
+    "@angular-devkit/build-angular": "14.0.2",
+    "@angular-eslint/eslint-plugin": "13.5.0",
+    "@angular-eslint/eslint-plugin-template": "13.5.0",
+    "@angular-eslint/template-parser": "13.5.0",
+    "@angular/cli": "14.0.2",
+    "@angular/compiler-cli": "14.0.2",
+    "@angular/language-service": "14.0.2",
+    "@nrwl/cli": "14.3.6",
+    "@nrwl/cypress": "14.3.6",
+    "@nrwl/eslint-plugin-nx": "14.3.6",
+    "@nrwl/jest": "14.3.6",
+    "@nrwl/linter": "14.3.6",
+    "@nrwl/nx-cloud": "14.1.2",
+    "@nrwl/workspace": "14.3.6",
     "@types/jest": "27.4.1",
     "@types/node": "16.11.7",
-    "@typescript-eslint/eslint-plugin": "5.24.0",
-    "@typescript-eslint/parser": "5.24.0",
+    "@typescript-eslint/eslint-plugin": "5.29.0",
+    "@typescript-eslint/parser": "5.29.0",
     "cypress": "9.1.0",
-    "eslint": "8.15.0",
-    "eslint-config-prettier": "8.1.0",
-    "eslint-plugin-cypress": "2.10.3",
+    "eslint": "8.18.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-cypress": "2.12.1",
     "jest": "27.5.1",
     "jest-preset-angular": "11.1.1",
-    "ng-packagr": "~14.0.0",
-    "nx": "14.2.0-rc.2",
-    "postcss": "^8.4.5",
+    "ng-packagr": "~14.0.2",
+    "nx": "14.3.6",
+    "postcss": "^8.4.14",
     "postcss-import": "~14.1.0",
-    "postcss-preset-env": "~7.5.0",
+    "postcss-preset-env": "~7.7.1",
     "postcss-url": "~10.1.3",
-    "prettier": "2.6.2",
+    "prettier": "2.7.1",
     "ts-jest": "27.1.4",
-    "ts-node": "10.8.0",
-    "typescript": "4.6.4"
+    "ts-node": "10.8.1",
+    "typescript": "4.7.4"
```

issue: Let's intentionally pin TypeScript to 4.6.x since that is the earliest TypeScript version officially supported by Angular 14.
```suggestion
    "typescript": "4.6.4"
```